test/tests/smoke_test.rs

```diff
@@ -302,7 +303,7 @@ fn smoke_test() {
     hasher.update(&vendor_pk_hash);
     hasher.update(&owner_pk_hash);
     hasher.update(/*ecc_vendor_pk_index=*/ &[0u8]); // No keys are revoked
-    hasher.update(/*lms_vendor_pk_index=*/ &[255u8]); // Because LMS is disabled
+    hasher.update(&[image.manifest.header.vendor_ecc_pub_key_idx as u8]);
```

Comment:
ecc_pub_key_idx? Should't this be LMS?

Reply:
The ROM implements this like this:

https://github.com/chipsalliance/caliptra-sw/blob/00b0503594a0ad7540b649ad93d8afbad797b8ae/rom/dev/src/flow/cold_reset/fmc_alias.rs#L159-L170

Reply:
It absolutely should.  I guess I managed to copy and paste the wrong one. Thanks for the catch!  I'll have a fix up in a minute.